.github/workflows/check.yml

```diff
@@ -56,6 +56,23 @@ jobs:
 
       - name: Run Tests
         run: bundle exec fastlane ${{ matrix.lane }}
+
+      - name: Check Static Analyzer Output
+        id: analyzer-output
+        run: |
+          if [[ -z $(find ./derived_data -name "report-*.html") ]]; then
```

Comment:
Shouldn't the search pattern used in this `find` (which is the same as the glob `./derived_data/**/report-*.html`) be consistent with the glob used in the "Static Analyzer Reports Uploading" step (`./derived_data/**/StaticAnalyzer/**/*.html`)?

(Of the two, probably better to use the more permissive one – a false positive is probably better than a false negative here, since at least they’ll be brought to our attention by a build failure, and we can then refine the glob if we discover false positives.)

Reply:
It might also be handy to add a comment to the script with an example of a path that you're expecting this glob to match.

Reply:
> probably better to use the more permissive one

Also, reading this again, I don't think there is a "more permissive one" – each is permissive in different ways.

Reply:
77f834ef2961f8f956ba93f65b144699d38e2d9e
I tried with less permissive with no luck (`find` spits that path doesn't exist). But more permissive works and all tests passed. Also it's obvious (just `/derived_data`), so is example comment there really needed?

Reply:
OK, don't worry about the comment then.